crates/network/src/native/client.rs

```diff
@@ -72,7 +76,7 @@ pub struct GameClientView {
     pub inner: Element,
 }
 
-impl ElementComponent for GameClientView {
+impl ElementComponent for ClientView {
```

Comment:
I wonder if this can be transitioned to a `#[element_component]`?

Reply:
It could, but will be remade completely using a schema and exposed to the guest code